### DISCUSSION TITLE
How to import beatmaps from other computer (osu!laser)

### DISCUSSION THREAD

**Discussion by pranshuthegamer** [ppy/osu#20500] [Q&A] [answered]
i have a laptop with 1gb of beatmaps, and i want to import them to my new laptop in osu laser.
copying the files over didnt work, then i saw that theres a database. is there a way to merge the databases?

**Comment by peppy** [comment, marked as answer]
Merging is not possible. You can copy the files over if you prefer to use that database, though.

**Reply by pranshuthegamer** [reply]
where is the database?

**Reply by pranshuthegamer** [reply]
i copied the files folder and nothing changed

**Reply by integerrr** [reply]
I don't believe copying just the `files` folder is enough as there are realm database files as well, please copy the entire `osu` folder
